python/lsst/ts/tunablelaser/mock_server.py

```diff
@@ -117,6 +131,152 @@ def __repr__(self):
         return f"{self.register_name}\n{self.register_id}\n{self.register_field}\n"
 
 
+class MockNT252:
+    """Implement the mock NT252 device.
+
+    Attributes
+    ----------
+    log : `logging.Logger`
+        The log.
+    wavelength : `int`
+        The wavelength.
+    temperature : `int`
+        The temperature.
+    propagating : `Power`
+        The propagation state of the laser.
+    propagation_mode : `Mode`
+        The propagation mode.
+    output : `Output`
+        The output energy level.
+    display_current : `int`
+        The display current.
+    burst_length : `int`
+        The burst length.
+    """
+
+    def __init__(self) -> None:
+        self.log = logging.getLogger(__name__)
+        self.wavelength = 650
```

Comment:
Please do not hardcode the values for wavelength, temperature and display_current but use random generators with ranges fit for those values. See the ts_ess_common package for an example.